Can't be imported under Jest

I'm trying to use this module in a `create-react-app` project, and I'm trying to test my project with the built-in Jest test system.

Jest doesn't run Node in ESM mode; it can handle ESM code with `import` statements, but behind the scenes it does transpilation to run on `require()`.

To use Jest with dependencies that are ESM modules, you have to add them to a negative regular expression in `jest.transformIgnorePatterns` in your `package.json`, so that Jest knows it needs to transpile them:

```
  "jest": {
    "transformIgnorePatterns": [
      "node_modules/(?!(@bjorn3/browser_wasi_shim)/)"
    ]
  }
```

But even though Jest can transform this module just fine, it can't deal with the conditional export in this project's `package.json`, which only specifies a file to expose in "import" mode:

https://github.com/bjorn3/browser_wasi_shim/blob/f1f5fd266484e0342fba6579bfc9bfc932d966eb/package.json#L25-L26

I get a Jest error like this when I try to run my tests:

```[anovak@swords sequenceTubeMap]% npm run test -- --watchAll=false src/GBZBaseAPI.test.js

> sequence-tube-maps@0.1.0 test
> react-scripts test --reporters default jest-compact-reporter --seed=1



Jest v27.5.1 node v18.7.0 darwin jest-min-reporter v0.1.0
Browserslist: caniuse-lite is outdated. Please run:
  npx update-browserslist-db@latest
  Why you should do it regularly: https://github.com/browserslist/update-db#readme
 FAIL  src/GBZBaseAPI.test.js
  ● Test suite failed to run

    Cannot find module '@bjorn3/browser_wasi_shim' from 'src/GBZBaseAPI.mjs'

    Require stack:
      src/GBZBaseAPI.mjs
      src/GBZBaseAPI.test.js

      1 | import { APIInterface } from "./APIInterface.mjs";
      2 | import { readFile } from "fs-extra";
    > 3 | import { WASI, File, OpenFile, PreopenDirectory } from "@bjorn3/browser_wasi_shim";
        | ^
      4 |
      5 | // TODO: The Webpack way to get the WASM would be something like:
      6 | //import QueryWasm from "gbz-base/target/wasm32-wasi/release/query.wasm";

      at Resolver.resolveModule (node_modules/jest-resolve/build/resolver.js:324:11)
      at Object.<anonymous> (src/GBZBaseAPI.mjs:3:1)
```

If I add a `"default": "./dist/index.js"` export to this project's `package.json` in my `node_modules`, Jest can find it and import and transpile it.

There might be a way to give Jest a [custom module resolver](https://github.com/jestjs/jest/pull/10393) to work around this, but I think the configuration fields required are not among those that Facebook permits you to use in a `create-react-app` project.

It would be good if this project exposed an export like "default" that would work in this situation.

--- a/package.json
+++ b/package.json
@@ -23,7 +23,8 @@
   "exports": {
     ".": {
       "types": "./typings/index.d.ts",
-      "import": "./dist/index.js"
+      "import": "./dist/index.js",
+      "default": "./dist/index.js"
     }
   },
   "typings": "./typings/index.d.ts",

